src/scripts/Drop-seq_alignment.sh

```diff
@@ -24,8 +24,6 @@
 . "$(dirname "$0")"/defs.sh
 
 outdir=$(pwd)
-genomedir=
```

Comment:
What do you mean by 'inherited from somewhere'? Can you give an example how I could overwrite a variable from outside the script?

Reply:
It could be in someone's rc file or an exported shell variable.  Some examples:
```
% cat print_genomedir.sh 
#! /bin/sh

echo "genomedir=$genomedir"
% ./print_genomedir.sh 
genomedir=
% export genomedir='Hello, World!'
% ./print_genomedir.sh 
genomedir=Hello, World!
% genomedir='Goodbye, Cruel World!' ./print_genomedir.sh 
genomedir=Goodbye, Cruel World!
```

Reply:
OK. Sure. I get it. AFAIK, such should be considered environment variables and therefore capitalized (`GENOMEDIR`, in this case).
But if you prefer to unset them, I can add a commit to do that consistently.
I'll take care of that tomorrow and convert this back from draft mode once it's ready.

Thank you for the careful review, constructive comments, and helpful explainations.

Reply:
I agree that environment variables should be capitalized, but people do weird things, so unsetting them would be good.  Thanks!